src/utils/auth/auth.provider.tsx

```diff
@@ -476,38 +466,42 @@ export function AuthRequestsProvider({
     });
   }, []);
 
-  const statusOverride = useStatusOverride();
```

Comment:
It was a bit convoluted how to get this value as this provider didn't have access to the router. I think the new version should be better, as it doesn't need this.